### ISSUE TITLE
Cannot patch the game

### ISSUE THREAD

**Issue by matepont** [isacucho/FnMacAssistant#10]
Operation is not permitted that's what the report says here:
Failed to patch the app: [Errno 1] Operation not permitted: '/Applications/Fortnite.app/Wrapper/FortniteClient-IOS-Shipping.app/embedded.mobileprovision'
it would be awesome if someone has a solution to this problem.
Thanks!

**Comment by matepont** [comment]
nevermind I patched it manually. If anyone has this problem and doesn't understand it just download the full zip file on github and from the files folder drag the embedded.mobileprovision file to >Application>Fortnite.app>Wrapper>FortniteClient.app

**Comment by isacucho** [comment]
Hi! This issue can be easily solved by going to settings > privacy and security > App Management and adding FnMacAssistant to that list. For some unknown reason not everyone needs to enable this setting, but some do. 

I saw you manually patched it and I’m glad you were able to solve it on your own, but I know that manually patching every 7 days can be a bit of a pain, that’s the reason this app exists in the first place. 

If you have any other questions or issues, you can follow up here or join the discord :)

**Comment by matepont** [comment]
Yes now that you are asking I do have a question and I just can't find an answer to this. So I got the patch the game starts very well but after the game trying to download 13 gig's update it somehow just freezes and never finishes. I try to log to find out what happened why it got stuck I asked chatgpt my little programmer to help me read a bit of code and it says that the app trying to access those parts of the macos system that is secured by SIP allegedly so yeah I'm kinda stuck here.

**Comment by isacucho** [comment]
Hi! I honestly don’t know what could be happening. Fortnite should work without disabling SIP. I have SIP enabled and am not having this issue. Are you sure it is not downloading? Sometimes the progress bar doesn’t move with the actual download. You could check how much it has downloaded by going to finder and searching ‘FortniteGame’, then selecting the heaviest folder, right clicking, and clicking on ‘show information’. 

It is weird because this has never happened before. Are you sure this logs are from Fortnite? Maybe you could try closing all other apps. Could you send me the log? Maybe I could find something in it.

**Comment by matepont** [comment]
Well i did not opened all the folders because theres so many but I think the heaviest one is like 15.59 gigs. Could you navigate me where can i find the logs for Fortnite I would be happy if you could take a look but I don't know where is the right one

**Comment by isacucho** [comment]
Huh, well now that’s weird. Maybe you can try reinstalling? You could delete all FortniteGame folders (both with lowercase and uppercase) and delete the app, then reinstall it. 

With the logs I meant what you asked ChatGPT. Or if it is easier for you you could send the conversation link as well.

**Comment by matepont** [comment]
I couldn't show you because the conversation is in my language so you wouldn't understand but I can tell you that I opened up terminal and put down this code: log show --predicate 'eventMessage contains "Fortnite"' --info
to show me info about what the program trying to do and why is it stuck and it loaded a bunch of log files but I uploaded a txt of the last few codes that I initially asked chatgpt to translate what it is
[log.odt](https://github.com/user-attachments/files/18216470/log.odt)

**Comment by matepont** [comment]
it seems that the download stopped again and I successfully located the log file and I get this error after every 30 seconds:
[2024.12.21-00.50.30:338][658]LogFortInstallBundleManager: Error: InstallBundleSourceBPS: BPT Watchdog Discovered Failure as request GFP_BRRoot timed out!
